### PR TITLE
Video call prompt

### LINKED ISSUE #2571: Nudge members to have video call before meeting
We had a discussion already about this on Slack I believe and now I got the same suggestion through support: add some info that hints members to make a video call before meeting. The member at support said it would be less uncomfortable when the platform suggests its, than asking for it themselves.

### PR DESCRIPTION
**Pull request by chagai95**
#### Proposed Changes

* Prompting the user to a video call before hosting

#### Testing Instructions

* click on the Yes, I can host! , you can see this message before sending your first message to someone in the messages layout

Fixes #2571

### CODE REVIEW

**Comment by noahsmindfuck** [comment]
What about moving it to the right and out of the message window? In the are of "is this convo in the spirit of trustroots" and in mobile just make a green camera symbol to klick.

**Review by noahsmindfuck — CHANGES_REQUESTED** [review]
What about moving it to the right and out of the message window? In the are of "is this convo in the spirit of trustroots" and in mobile just make a green camera symbol to klick.

Also: why is the whatsapp me in this commit?

**Comment by chagai95** [comment]
> What about moving it to the right and out of the message window? In the are of "is this convo in the spirit of trustroots" and in mobile just make a green camera symbol to klick.
> 
What would be the purpose of that? It'll be much less visual and I didn't quite understand what you mean with the camera icon... The users need to be aware this is not just a video call button but a room they would both have to enter at the same time, which is why we need text to explain it.... This is unfortunately not a video calling feature that we are adding to the platform, that'll be a little bit more complicated...


> Also: why is the whatsapp me in this commit?

The WhatsApp is there because I didn't discard the changes, weird I didn't think VSC would push that, thanks for noticing!

**Comment by chagai95** [comment]
Hey @simison do you think this is something you could merge as is, it's been stagnating for a while and I think it might be better on production the way it is as opposed to not at all?

**Comment by sitarane** [comment]
Are we even allowed to integrate jitsi's server in an app like that? Did anyone check the jitsi fine prints?

Jitsi is nice and all to let people use it's infra for free, but there probably are limits...

**Comment by chagai95** [comment]
I'm pretty sure it's allowed (otherwise we can use any other free jitsi instance out there or run our own). If we are going to merge this then I'll personally communicate this to the jitsi team and maybe they'll put us on their website as an open source community project which uses jitsi, then we'll definitely be in the clear, (otherwise we can remove it)

Also I don't know how integrated jitsi really is in this case we're basically just generation a random URL and telling people to meet...

File: modules/users/client/views/profile/profile-view.client.view.html [hunk 1]
```diff
@@ -155,6 +155,14 @@
                   Send a message
                 </a>
               </li>
+              <li ng-if="app.appSettings.referencesEnabled">
+                <a
+                  ui-sref="profile.experiences.new({username: profileCtrl.profile.username})"
+                >
+                  <i class="fa icon-message-alt"></i>
+                  Contact me via Phone
```
Comment (sitarane) [inline modules/users/client/views/profile/profile-view.client.view.html:163]: Hard coded string? Shouldn't we use i18n keys?

File: modules/messages/client/components/QuickReply.js [hunk 2]
```diff
@@ -44,11 +44,36 @@ export default function QuickReply({ onSend, onFocus }) {
...
+              const characters =
+                'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+              const charactersLength = characters.length;
+              for (let i = 0; i < 15; i++) {
+                result += characters.charAt(
+                  Math.floor(Math.random() * charactersLength),
+                );
+              }
+              alert(result);
+              const jitsi_random = result;
```
Comment (sitarane) [inline modules/messages/client/components/QuickReply.js:58]: This risk intersecting with an existing jitsi room. I bet there's a way to just query jitsi for a random room and use that.

File: modules/users/client/views/profile/profile-view.client.view.html (hunk 1 above)
Comment (chagai95) [reply modules/users/client/views/profile/profile-view.client.view.html:163]: We should

**Pull request closed without merging**